aiida_quantumespresso/calculations/cp.py

```diff
@@ -30,97 +34,176 @@ class CpCalculation(BasePwCpInputGenerator, CalcJob):
     For more information, refer to http://www.quantum-espresso.org/
     """
 
+    # Constants to use in the calculation
     _CP_READ_UNIT_NUMBER = 50
     _CP_WRITE_UNIT_NUMBER = 51
+    _FILE_XML_PRINT_COUNTER_BASENAME = "print_counter.xml"
+    _FILE_XML_PRINT_COUNTER = os.path.join(
+        BasePwCpInputGenerator._OUTPUT_SUBFOLDER,
+        "{}_{}.save".format(BasePwCpInputGenerator._PREFIX, _CP_WRITE_UNIT_NUMBER),
+        _FILE_XML_PRINT_COUNTER_BASENAME,
+    )
+
+    # Input file "sections" that we are going to write by calculation type
+    # The term namelist is part of FORTRAN's jargon
+    _automatic_namelists = {
+        "scf": ["CONTROL", "SYSTEM", "ELECTRONS"],
```

Comment:
Even though not a hard convention, the custom is to use single quotes for string literal identifiers such as keys in dictionaries.